includes/modules/payment/paypalwpp.php

```diff
@@ -306,7 +306,7 @@ function process_button() {
         });
       };
       </script>
-      <script src=\"//www.paypalobjects.com/api/checkout.js\" async></script>";
+      <div data-paypal-button=\"yes\" data-paypal-id=\"" . MODULE_PAYMENT_PAYPALWPP_MERCHANTID . "\"></div>"
```

Comment:
Yes, in my original code it was on the next line with a lot of commented out code between the end of that line and the end of the one below.

Added back to the end of that line.